docs/overview.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Overview
+description: This section provides a conceptual overview of Octopus Deploy.
+position: 0
+---
+
+Welcome!
+
+This section provides a conceptual overview of Octopus Deploy.
+
+As an Octopus user, you define the process for deploying your software. You specify the environments the software is deployed to and who on your team can deploy to which environments. For instance, you might want developers to deploy to the Dev environment, but not Test or Production, you might want the QA team to deploy to the Test environment, but not Dev or Production. Taking this approach means that even if different members of the team trigger deployments, the deployment process remains consistent and predictable.
+
+When a release of your software is created in Octopus, you can deploy it as many times as needed.
+
+## The Octopus Deploy Server
+
+You can install your own [self-hosted](#self-hosted-octopus) instance of the Octopus Deploy Server or use [Octopus Cloud](#octopus-cloud).
+
+## Self-Hosted Octopus {#self-hosted-octopus}
+
+Installing the self-hosted [Octopus Deploy server](/docs/installation/index.md) configures the [Octopus Web Portal](/docs/getting-started/index.md#the-octopus-web-portal) and the [Octopus REST API](/docs/octopus-rest-api/index.md). The [installation documentation](/docs/installation/index.md) provides instructions for downloading, installing, and configuring your Octopus Deploy server.
+
+### Octopus Cloud {#octopus-cloud}
+
+Octopus Cloud is the hosted version of Octopus Deploy. We designed Octopus Cloud and self-hosted Octopus to provide the same functionality; however, there are some minor differences, for instance, with Octopus Cloud, we're [responsible](/docs/administration/security/index.md#responsibility) for taking backups, upgrading the service, and maintaining and monitoring the underlying systems.
```

Comment:
Not sure on the grammar here: "; however, there are some minor differences, for instance,"

Consider "; however, there are some minor differences. For instance, ..."